docs/install/upgrade.md

```diff
@@ -50,3 +72,5 @@ from Winget.
 ```pwsh
 winget install Coder.Coder
 ```
+
+</div>
```

Comment:
I think this was an accident to add since this is just Markdown.

Reply:
Thanks! this is (closing) our tabs component. Makes it look like this:

![image](https://github.com/user-attachments/assets/907143b1-bdc4-4431-8887-379bb96e96ae)

[preview link](https://coder.com/docs/@upgrade-tabs/install/upgrade#other-methods)

Reply:
oh interesting!